### ISSUE TITLE
warning from numpy

### ISSUE THREAD

**Issue by jingpengw** [seung-lab/connected-components-3d#69]
this is not a big issue, but it might create problems in the future.
I just would like to keep a mark here, nothing urgent to fix.

```
Processing connected-components-3d-3.2.0.tar.gz
Writing /var/folders/gc/b6s140td6xdbfxdrsmkjtww80001lg/T/easy_install-wc89qksb/connected-components-3d-3.2.0/setup.cfg
Running connected-components-3d-3.2.0/setup.py -q bdist_egg --dist-dir /var/folders/gc/b6s140td6xdbfxdrsmkjtww80001lg/T/easy_install-wc89qksb/connected-components-3d-3.2.0/egg-di
st-tmp-c5t0i0lp
In file included from cc3d.cpp:645:
In file included from /Users/jwu/opt/anaconda3/envs/wasp/lib/python3.7/site-packages/numpy/core/include/numpy/arrayobject.h:4:
In file included from /Users/jwu/opt/anaconda3/envs/wasp/lib/python3.7/site-packages/numpy/core/include/numpy/ndarrayobject.h:12:
In file included from /Users/jwu/opt/anaconda3/envs/wasp/lib/python3.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1944:
/Users/jwu/opt/anaconda3/envs/wasp/lib/python3.7/site-packages/numpy/core/include/numpy/npy_1_7_deprecated_api.h:17:2: warning: "Using deprecated NumPy API, disable it with "
      "#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-W#warnings]
#warning "Using deprecated NumPy API, disable it with " \
 ^
1 warning generated.
```

**Comment by william-silversmith** [comment]
Unfortunately, this is a known issue with cython. They claim it will be
fixed in Cython 3 (the next release after 0.29) which should be released
soon.

**Comment by william-silversmith** [comment]
Closing until cython 3.0 comes out.
